### PR TITLE
Version update and documentation updates and some minor fixes

### PR DESCRIPTION
**Pull request by sarthakpati**
Changes:

- References updated
- Version updated (and associated HTML updates)
- Default save location for FE updated as suggested by @eac-upenn in https://github.com/CBICA/CaPTk/issues/734#issuecomment-652466940
- For PSP, the "new" modality (T1-T1CE) construction has been standardized for testing and training. Previously, they were different, which caused issues during testing. This was spotted and rectified as per direction from @chiharusako

### CODE REVIEW

**Comment by AlexanderGetka-cbica** [comment]
Looks like GitHub has picked up the new run this time.

**Comment by sarthakpati** [comment]
> Looks like GitHub has picked up the new run this time.

I pushed a dummy commit. 😄

File: src/applications/PseudoProgressionEstimator.cpp [hunk 1]
```diff
@@ -1368,7 +1368,7 @@ VariableSizeMatrixType PseudoProgressionEstimator::LoadPseudoProgressionTraining
     ImageType::Pointer OriginalT1ImagePointer = ReadNiftiImage<ImageType>(static_cast<std::string>(currentsubject[CAPTK::ImageModalityType::IMAGE_TYPE_T1]));
     ImageType::Pointer OriginalT2ImagePointer = ReadNiftiImage<ImageType>(static_cast<std::string>(currentsubject[CAPTK::ImageModalityType::IMAGE_TYPE_T2]));
 
-    ImageType::Pointer OriginalT1T1CEImagePointer = MakeAdditionalModality<ImageType>(OriginalT1ImagePointer, OriginalT1CEImagePointer);
+    ImageType::Pointer OriginalT1T1CEImagePointer = MakeAdditionalModality<ImageType>(OriginalT1CEImagePointer, OriginalT1ImagePointer);
```
Comment (ashishsingh18) [inline src/applications/PseudoProgressionEstimator.cpp:1371]: Why the swap of OriginalT1CEImagePointer, OriginalT1ImagePointer ?

Comment (sarthakpati) [reply src/applications/PseudoProgressionEstimator.cpp:1371]: For consistency between training and testing, as I have mentioned in the PR summary.

**Pull request merged**